reporting_app/static/stage_timeline.js

```diff
@@ -0,0 +1,57 @@
+
+function _to_moment(str_date) {
+    return moment(str_date, 'DD_MM_YYYY_HH:mm:SS');
+}
+
+
+function _reformat(date) {
+    return _to_moment(date).format('YYYY-MM-DD HH:mm:ss');
+}
+
+
+function build_tooltip(stage) {
+    var t = '<div><b>' + stage.stage_name + '</b><br/>Started: ' + _reformat(stage.date_started) + '<br/>';
+    if (stage.date_finished) {
+        t += 'Finished: ' + _reformat(stage.date_finished) + '<br/>';
+        t += 'Exit status: ' + stage.exit_status + '<br/>';
+    }
+    t += '</div>';
+    return t;
+}
+
+
+function stage_timeline(div_id, proc) {
+    var stages = proc.stages.map(
```

Comment:
Should test that you have stages here as some analysis_driver_proc do not have any